ci/test/04_install.sh

```diff
@@ -50,34 +51,39 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
                   --env-file /tmp/env \
                   --name $CONTAINER_NAME \
                   $DOCKER_NAME_TAG)
-  export DOCKER_CI_CMD_PREFIX="docker exec $DOCKER_ID"
+  export DOCKER_CI_CMD_PREFIX_ROOT="docker exec -u 0 $DOCKER_ID"
+  export DOCKER_CI_CMD_PREFIX="docker exec -u $(id -u) $DOCKER_ID"
 else
   echo "Running on host system without docker wrapper"
 fi
 
 CI_EXEC () {
   $DOCKER_CI_CMD_PREFIX bash -c "export PATH=$BASE_SCRATCH_DIR/bins/:\$PATH && cd \"$P_CI_DIR\" && $*"
 }
+CI_EXEC_ROOT () {
+  $DOCKER_CI_CMD_PREFIX_ROOT bash -c "export PATH=$BASE_SCRATCH_DIR/bins/:\$PATH && cd \"$P_CI_DIR\" && $*"
+}
 export -f CI_EXEC
+export -f CI_EXEC_ROOT
```

Comment:
what about removing this and using `sudo`? I guess sudo isn't installed?

Reply:
sudo isn't installed in the container and it's generally not recommended to install or use it inside docker. I renamed the variable here to make it explicit, but this is the same behavior as before: if we don't specify a user (`docker exec -u <some user>`), docker defaults to running root inside the container. The thing that has changed is now when we run `CI_EXEC`, we are running as our newly created non-root user.